server/models.py

```diff
@@ -84,6 +84,11 @@ class Budget(BaseModel):
 
     user_id = db.Column(db.Integer, db.ForeignKey('users.id'))
 
+    def to_dict(self, visited=None):
```

Comment:
updates to_dict method for budgets so it is not sending all user data with each budget